cms/djangoapps/contentstore/management/commands/reset_course_content.py

```diff
@@ -0,0 +1,57 @@
+"""
+Django management command to reset the content of a course to a a different
+version, as specified by an ObjectId from the DraftVersioningModulestore (aka Split).
+"""
+from textwrap import dedent
+
+from django.core.management import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+
+from xmodule.modulestore.django import modulestore
+
+
+class Command(BaseCommand):
+    """
+    Reset the content of a course run to a different version, and publish.
+
+    This is a powerful command; use with care.
+    It's analogous to `git reset --hard VERSION && git push -f`.
+
+    The intent of this is to restore overwritten course content that has not yet been
+    pruned from the modulestore. I guess you could use it to change a course's content
+    to any structure in Split you wanted, though.
+
+    Make sure you have validated the value of `course_id` and `version_guid`.
+    There is no confirmation prompt.
+
+    Example:
+
+        ./manage.py reset_course_content "course-v1:my+cool+course" "5fb5772e2fe4c7c76493c241"
```

Comment:
@ormsbee, I know you mentioned enabling this via Django admin somehow. I agree it would be great if we didn't need SRE to run this for us.

I'm not sure I'd go about implementing that, though. It seems that Django only supports admin pages that are associated with a particular model.

Maybe we choose a related-seeming model, and tack this command on to it? It feels fishy, but we could make it an "Action" on Course Overview.

Reply:
Yeah, come to think on it, you _can_ do it, but it's more involved and includes putting a primitive view. Doing this management command in the short term is better.

Reply:
Basically, you do this: https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_urls